packages/core-components/api-report-alpha.md

```diff
@@ -0,0 +1,73 @@
+## API Report File for "@backstage/core-components"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { TranslationRef } from '@backstage/core-plugin-api/alpha';
+
+// @alpha (undocumented)
+export const coreComponentsTranslationRef: TranslationRef<
+  'core-components',
+  {
+    readonly 'link.openNewWindow': 'Opens in a new window';
+    readonly 'table.filter.title': 'Filters';
+    readonly 'table.filter.clearAll': 'Clear all';
+    readonly 'signIn.title': 'Sign In';
+    readonly 'signIn.loginFailed': 'Login failed';
+    readonly 'signIn.customProvider.title': 'Custom User';
+    readonly 'signIn.customProvider.subtitle2': 'This selection will not be stored.';
+    readonly 'signIn.customProvider.subtitle': 'Enter your own User ID and credentials.';
+    readonly 'signIn.customProvider.userId': 'User ID';
+    readonly 'signIn.customProvider.tokenInvalid': 'Token is not a valid OpenID Connect JWT Token';
+    readonly 'signIn.customProvider.continue': 'Continue';
+    readonly 'signIn.customProvider.idToken': 'ID Token (optional)';
+    readonly 'signIn.guestProvider.title': 'Guest';
+    readonly 'signIn.guestProvider.description': 'You will not have a verified identity, meaning some features might be unavailable.';
+    readonly 'signIn.guestProvider.enter': 'Enter';
+    readonly 'signIn.guestProvider.subtitle': 'Enter as a Guest User.';
+    readonly 'sidebar.shipToContent': 'Skip to content';
+    readonly 'sidebar.starredIntroText': 'Fun fact! As you explore all the awesome plugins in Backstage, you can actually pin them to this side nav.Keep an eye out for the little star icon (⭐) next to the plugin name and give it a click!';
```

Comment:
Think we can skip this, and `sidebar.recentlyViewedIntroText` as we're going to clean them up, they're really not used, and should belong elsewhere rather than `core-components`

Reply:
ok